packages/common/src/allo/backends/allo-v2.ts

```diff
@@ -1340,3 +1352,40 @@ export type ProjectWithMerkleProof = {
   amount: BigNumber;
   merkleProof: string[];
 };
+
+/**
+ * Generate merkle tree
+ *
+ * To get merkle Proof: tree.getProof(distributions[0]);
+ * @param matchingResults MatchingStatsData[]
+ * @returns
+ */
+export const generateMerkleTreeV2 = (
```

Comment:
do we still need the generateMerkleTree function in utils, if we have one in allo-v1.ts and another one in allo-v2.ts?

Reply:
@0xKurt  You are right. We don't need the one in utils. I've removed it now.